sway/input/seat.c

```diff
@@ -722,17 +723,66 @@ void seat_remove_device(struct sway_seat *seat,
 	seat_update_capabilities(seat);
 }
 
+static bool xcursor_manager_is_named(const struct wlr_xcursor_manager *manager,
+                                     const char *name) {
+	return (!manager->name && !name) ||
+		(name && manager->name && strcmp(name, manager->name) == 0);
+}
+
 void seat_configure_xcursor(struct sway_seat *seat) {
-	// TODO configure theme and size
+	int cursor_size = 24;
 	const char *cursor_theme = NULL;
 
-	if (!seat->cursor->xcursor_manager) {
-		seat->cursor->xcursor_manager =
-			wlr_xcursor_manager_create(cursor_theme, 24);
-		if (sway_assert(seat->cursor->xcursor_manager,
-					"Cannot create XCursor manager for theme")) {
-			return;
+	const struct seat_config *seat_config = seat_get_config(seat);
```

Comment:
`seat_get_config` will not match the wildcard config so you'll need to add
```c
if (!seat_config) {
    seat_config = seat_get_config_by_name("*");
}
```

Reply:
When testing, I noticed that the first time we came here, the default seat had no configuration. However, it was soon called again, this time with a `seat0` configuration available (with the `*` configuration applied/merged).

Out of curiosity (and so that I can learn); is that something that is specific to the default seat? Or what is the reason for explicitly asking for the `*` config?

Reply:
Anyway, I've updated it to also try the `*` config. Resolving conversation for now.

Reply:
After thinking about it, I think there is only one case where a seat would not have an explicit seat config and can fallback to the wildcard seat config.

If you had the following in your config (and no `seat seat0 <seat-subcommand>` lines):
```
seat seat1 fallback true
seat * xcursor_theme redglass 16
```
Then, there would be no seat config created for `seat0`. `seat1` has an explicit `fallback true` so there would not be an implicit one added for `seat0`. `seat0` would just exist as the default seat, but have no attachments and no config.

Any seat other than `seat0`, would require some `seat <name> <seat-subcommand>` to be executed, which would create a seat config before the seat itself.